pom.xml

```diff
@@ -0,0 +1,245 @@
+        <aspectj.version>1.9.6</aspectj.version>
+        <slf4j.version>1.7.32</slf4j.version>
+        <cucumber.version>7.0.0</cucumber.version>
+        <lombok.version>1.18.22</lombok.version>
+        <jdi-light-html.version>1.3.17</jdi-light-html.version>
+
+        <!--        plagins versions-->
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <allure-maven.version>2.10.0</allure-maven.version>
+        <allure.link.issue.pattern>https://example.org/browse/{}</allure.link.issue.pattern>
+        <allure.link.tms.pattern>https://example.org/browse/{}</allure.link.tms.pattern>
+
+        <!--        path -->
+        <testng.suite.path>${project.basedir}/src/test/resources/suits</testng.suite.path>
+        <lib.calculator.path>${project.basedir}/lib/calculator-1.0.jar</lib.calculator.path>
+
+    </properties>
+
+<dependencies>
+    <dependency>
+        <groupId>com.epam.tat.module4</groupId>
+        <artifactId>calculator</artifactId>
+        <version>${calculator.version}</version>
+        <scope>system</scope>
+        <systemPath>${lib.calculator.path}</systemPath>
+    </dependency>
+    <dependency>
+        <groupId>org.testng</groupId>
+        <artifactId>testng</artifactId>
+        <version>${testng.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.seleniumhq.selenium</groupId>
+        <artifactId>selenium-java</artifactId>
+        <version>${selenium-java.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>io.github.bonigarcia</groupId>
+        <artifactId>webdrivermanager</artifactId>
+        <version>${webdrivermanager.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.assertj</groupId>
+        <artifactId>assertj-core</artifactId>
+        <version>${assertj-core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>commons-configuration</groupId>
+        <artifactId>commons-configuration</artifactId>
+        <version>${commons-configuration.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>ru.yandex.qatools.htmlelements</groupId>
+        <artifactId>htmlelements-java</artifactId>
+        <version>${htmlelements-java.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>ru.yandex.qatools.htmlelements</groupId>
+        <artifactId>htmlelements-matchers</artifactId>
+        <version>${htmlelements-matchers.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>io.qameta.allure</groupId>
+        <artifactId>allure-testng</artifactId>
+        <version>${allure-testng.version}</version>
+        <scope>test</scope>
+    </dependency>
+<!--    <dependency>-->
+<!--        <groupId>org.slf4j</groupId>-->
+<!--        <artifactId>slf4j-simple</artifactId>-->
+<!--        <version>${slf4j.version}</version>-->
+<!--    </dependency>-->
```

Comment:
Let's remove commented dependency if we don't need it

Reply:
removed